lib/al/Library/Area/AreaObj.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <math/seadMatrix.h>
+#include <math/seadVector.h>
+#include "al/Library/HostIO/HioNode.h"
+#include "al/Library/Layout/LayoutActor.h"
```

Comment:
useless include

Reply:
should've been SceneObjHolder.h whoopsie